DS2450 Update onewire.markdown

moritzthecat opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
Added DS2450 in docs as needed for pull request.

<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/136882

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Documentation**
	- Added documentation for the DS2450 sensor device in the OneWire integration.
	- Included details about the DS2450's family code, device link, and voltage measurement capabilities.
	- Updated naming instructions for API connections in the Airthings integration documentation, clarifying character limits.
	- Removed the "Vehicle sleep" section from the Teslemetry integration documentation.
	- Made minor textual corrections and formatting adjustments in the voice control documentation for clarity and consistency.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

c0ffeeca7 committed:
Assist customize responses: Fix link (#37187)

moritzthecat committed:
DS2450 Update onewire.markdown

Added DS2450 in docs.

moritzthecat committed:
Merge branch 'next' into patch-1

epenet commented on source/voice_control/custom_sentences.markdown:
```diff
@@ -9,11 +9,13 @@ related:
     title: Sentence triggers
   - docs: /docs/automation/trigger/#sentence-wildcards
     title: Sentence wildcards
+  - docs: voice_control/custom_sentences_yaml
```
You seem to have picked up unrelated changes
Please rebase

epenet replied:
Still needs to be rebased
If you are using a browser to create the PR, it might be easier to simply close and reopen a new PR - ensuring that you select the `next` branch before you attempt to edit a file

moritzthecat replied:
ok

Bre77 committed:
Remove sleep section from Teslemetry (#37201)

dependabot[bot] committed:
build(deps-dev): bump jekyll from 4.4.0 to 4.4.1 (#37200)

Bumps [jekyll](https://github.com/jekyll/jekyll) from 4.4.0 to 4.4.1.
- [Release notes](https://github.com/jekyll/jekyll/releases)
- [Changelog](https://github.com/jekyll/jekyll/blob/master/History.markdown)
- [Commits](https://github.com/jekyll/jekyll/compare/v4.4.0...v4.4.1)

---
updated-dependencies:
- dependency-name: jekyll
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
AI personality: fix typo (#37203)

kaovilai committed:
airthings suggested API name too long (#37192)

* airthings suggested API name too long

* Update source/_integrations/airthings.markdown

* Update source/_integrations/airthings.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

---------

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

moritzthecat committed:
Merge branch 'next' into patch-1

moritzthecat committed:
Merge branch 'home-assistant:current' into patch-1

moritzthecat commented:
closed as suggested by @epenet https://github.com/home-assistant/home-assistant.io/pull/37199#discussion_r1935366814
